helm-charts/charts/warehouse/values.yaml

```diff
@@ -0,0 +1,207 @@
+# Ensure that the warehouse instance name differs from the starrocks instance name to avoid conflicts. For example, both
+# will generate configmaps using the 'nameOverride' as a prefix to the configmap name, followed by a specific suffix.
+nameOverride: "kube-starrocks"
+
+# TimeZone is used to set the environment variable TZ for pod, with Asia/Shanghai as the default.
+timeZone: Asia/Shanghai
+
+# This configuration is used to modify the root password during initial deployment.
+# After deployment is completed, it won't take effect to modify the password here and to do a `helm upgrade`.
+# It also supports providing secret name that contains password, using the password in the secret instead of the plaintext in the values.yaml.
+# When both password and passwordSecret are set, only passwordSecret takes effect.
+initPassword:
```

Comment:
this section now makes no sense, since the warehouse will not change the FE's root password, why does it needs the initPasswordSecret.

Think about if possible retrieve the related password configuration from the crd?

Also can we create a branch for this warehouse development, so it can be developed in a series of PRs instead of a big one. After tested stable enough, merge it back to main branch and do the big release.

Reply:
Good Suggestion!

1. You are right, the Warehouse do not need the initPassword Section.
2. so I create a main-warehouse branch first?

Reply:
create `feature/warehouse` branch